core-java-modules/core-java-streams-4/src/test/java/splitting/StreamSplittingUnitTest.java

```diff
@@ -0,0 +1,86 @@
+package splitting;
+
+import com.google.common.collect.Lists;
+import org.junit.Assert;
+import org.junit.Test;
+import rx.Observable;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Map;
+import java.util.stream.Collectors;
+
+public class StreamSplittingUnitTest {
+    // given
```

Comment:
Remove these comments. Partly because Baeldung code doesn't have random comments in. Partly because these comments don't say anything. See this article - https://codingcraftsman.wordpress.com/2019/07/19/lets-add-cruft-to-our-tests/ - warning it's quite an opinionated article, written as though I'm angry... read it as though I'm smiling while I point out how we've been taught to do weird things in our code.

Reply:
Ok, that's a take I didn't hear before. I removed given/when/then comments.